view/frontend/web/template/payment/offsite-installment-form.html

```diff
@@ -7,7 +7,7 @@
                checked: isChecked,
                click: selectPaymentMethod,
                visible: isRadioButtonVisible(),
-               enable: isActive() && !orderValueTooLow()" />
+               enable: isActive() && !orderValueTooLow(3000)" />
```

Comment:
It is not ideal, but setting it here allows to remove entire function.

Reply:
How about just re-use that constant, the one we declared in each payment JS file? 
Same as what we did for `restrictedToCurrencies: ['thb']`. Maybe something like `restrictedMinimumAmount`.
Just idea, would this work? 👇  : ) 
```js
orderValueTooLow: function () {
    return this.restrictedMinimumAmount && this.getTotal() < this.restrictedMinimumAmount;
},
```

Reply:
Agree - should be using a constant